.travis.yml

```diff
@@ -9,12 +9,11 @@ php:
 env:
   global:
       - ALGOLIA_APP_ID=K7MLRQH1JG
-      - ALGOLIA_API_KEY=Yzk5OWJkMjNmYWNlMTYwNDY5MjZhODBhMDRmODBmMzBlYzhiZTNhMDY0OWExM2I1ZTc3ZDRkYjcyZTBkMjE3YXJlc3RyaWN0U291cmNlcz0lNUIlMjI4LjM0LjIwOC4wJTVDJTJGMjAlMjIlMkMlMjI4LjM1LjE5Mi4wJTVDJTJGMjElMjIlMkMlMjI4LjM1LjIwMC4wJTVDJTJGMjMlMjIlMkMlMjIyMy4yMzYuNDguMCU1QyUyRjIwJTIyJTJDJTIyMjMuMjUxLjEyOC4wJTVDJTJGMTklMjIlMkMlMjIzNS4xODQuMC4wJTVDJTJGMTQlMjIlMkMlMjIzNS4xODguMC4wJTVDJTJGMTUlMjIlMkMlMjIzNS4xOTAuMC4wJTVDJTJGMTclMjIlMkMlMjIzNS4xOTAuMTI4LjAlNUMlMkYxOCUyMiUyQyUyMjM1LjE5MC4xOTIuMCU1QyUyRjE5JTIyJTJDJTIyMzUuMTkwLjIyNC4wJTVDJTJGMjAlMjIlMkMlMjIzNS4xOTIuMC4wJTVDJTJGMTQlMjIlMkMlMjIzNS4xOTYuMC4wJTVDJTJGMTUlMjIlMkMlMjIzNS4xOTguMC4wJTVDJTJGMTYlMjIlMkMlMjIzNS4xOTkuMC4wJTVDJTJGMTclMjIlMkMlMjIzNS4xOTkuMTI4LjAlNUMlMkYxOCUyMiUyQyUyMjM1LjIwMC4wLjAlNUMlMkYxNSUyMiUyQyUyMjM1LjIwMi4wLjAlNUMlMkYxNiUyMiUyQyUyMjM1LjIwMy4wLjAlNUMlMkYxNyUyMiUyQyUyMjM1LjIwMy4xMjguMCU1QyUyRjE4JTIyJTJDJTIyMzUuMjAzLjI0MC4wJTVDJTJGMjAlMjIlMkMlMjIzNS4yMDQuMC4wJTVDJTJGMTUlMjIlMkMlMjIzNS4yMDYuNjQuMCU1QyUyRjE4JTIyJTJDJTIyMzUuMjI0LjAuMCU1QyUyRjE0JTIyJTJDJTIyMzUuMjI4LjAuMCU1QyUyRjE2JTIyJTJDJTIyMTA0LjE1NC4wLjAlNUMlMkYxNSUyMiUyQyUyMjEwNC4xOTYuMC4wJTVDJTJGMTQlMjIlMkMlMjIxMDcuMTY3LjE2MC4wJTVDJTJGMTklMjIlMkMlMjIxMDcuMTc4LjE5Mi4wJTVDJTJGMTglMjIlMkMlMjIxMDguNTkuODAuMCU1QyUyRjIwJTIyJTJDJTIyMTA4LjE3MC4xOTIuMCU1QyUyRjIwJTIyJTJDJTIyMTA4LjE3MC4yMDguMCU1QyUyRjIxJTIyJTJDJTIyMTA4LjE3MC4yMTYuMCU1QyUyRjIyJTIyJTJDJTIyMTA4LjE3MC4yMjAuMCU1QyUyRjIzJTIyJTJDJTIyMTA4LjE3MC4yMjIuMCU1QyUyRjI0JTIyJTJDJTIyMTMwLjIxMS40LjAlNUMlMkYyMiUyMiUyQyUyMjEzMC4yMTEuOC4wJTVDJTJGMjElMjIlMkMlMjIxMzAuMjExLjE2LjAlNUMlMkYyMCUyMiUyQyUyMjEzMC4yMTEuMzIuMCU1QyUyRjE5JTIyJTJDJTIyMTMwLjIxMS42NC4wJTVDJTJGMTglMjIlMkMlMjIxMzAuMjExLjEyOC4wJTVDJTJGMTclMjIlMkMlMjIxNDYuMTQ4LjIuMCU1QyUyRjIzJTIyJTJDJTIyMTQ2LjE0OC40LjAlNUMlMkYyMiUyMiUyQyUyMjE0Ni4xNDguOC4wJTVDJTJGMjElMjIlMkMlMjIxNDYuMTQ4LjE2LjAlNUMlMkYyMCUyMiUyQyUyMjE0Ni4xNDguMzIuMCU1QyUyRjE5JTIyJTJDJTIyMTQ2LjE0OC42NC4wJTVDJTJGMTglMjIlMkMlMjIxNjIuMjE2LjE0OC4wJTVDJTJGMjIlMjIlMkMlMjIxNjIuMjIyLjE3Ni4wJTVDJTJGMjElMjIlMkMlMjIxNzMuMjU1LjExMi4wJTVDJTJGMjAlMjIlMkMlMjIxOTIuMTU4LjI4LjAlNUMlMkYyMiUyMiUyQyUyMjE5OS4xOTIuMTEyLjAlNUMlMkYyMiUyMiUyQyUyMjE5OS4yMjMuMjMyLjAlNUMlMkYyMiUyMiUyQyUyMjE5OS4yMjMuMjM2LjAlNUMlMkYyMyUyMiUyQyUyMjIwOC42OC4xMDguMCU1QyUyRjIzJTIyJTVE
-      # This API keys only works on travis
-      # has limited ACLs and other various restrictions
+      - secure: EH081aVVAcsF0+vpllXs78xjG5cS7QHUBAtZoIfPaLhlkViKK7NSZU7Xbdo+dKGkJIfcNgHmmZwYiaxg0S47DNSrSZbgMaWTDq7jplvh6M0QPf31QH1J0IGudvgP2LdnOsjZQbf7MSI0SOm98BALB1OfI0J5RBl+qUlw/3J4XoI=
```

Comment:
Using secure brings back the old problem of forked builds not running through because secure env variables are not available in forked builds. This makes it very difficult for contributors to see whether their changes result in a stable build and put additional burden on maintainers (all pull requests from forks must be pushed to the original repository for travis to test them).
What's the harm having an API key that only works on travis?

Reply:
Yes @alcaeus we will fix it with @julienbourdeau approach pretty soon.
We don't have the infra to solve it but we will.
Here we just move forward a bit, but we still really want to make community PR's pass the tests.
Thank you for pushing for it.

Reply:
That sounds great, thank you!